sample_files/.gitignore

```diff
@@ -36,10 +40,23 @@ coverage.xml
 
 # Translations
 *.mo
+*.pot
 
 # Pycharm
 .idea
 
+# Eclipse
+.settings
+
+# Visual Studio cache/options directory
+.vs/
```

Comment:
Shouldn't this be .vscode?

Reply:
https://github.com/OCA/maintainer-tools/pull/316

Reply:
Yeah, but now see what my colleague @Yajo added in other PR: https://github.com/OCA/maintainer-quality-tools/pull/519/files#diff-b1d6df5420097c477a0feb225b280b64R42

Reply:
Looks like both are valid. One is project folder for Visual Studio, the other is Visual Studio Code. Ambiguous naming, but different things.

Can we add both please?

Reply:
Done